internal/app/api/types/pagesort.go

```diff
@@ -0,0 +1,37 @@
+// Pagination and Sorting
+package types
+
+import "strconv"
+
+type PagingNSorting struct {
+	Size int `json:"size" default:"100"`
+	Page int `json:"page" default:"1"`
+}
+
+func (ps *PagingNSorting) Init(page, size string) error {
+
+	if size == "" && page == "" {
+		ps.Size = 100
+		ps.Page = 1
+		return nil
+	}
+
+	sizeInt, err := strconv.ParseInt(size, 10, 64)
```

Comment:
What if size has a value, but page not?

Reply:
the default value of page is 1 (logic in line 21)
![image](https://user-images.githubusercontent.com/48717211/124688981-e54e3000-df01-11eb-94dd-a2ae4cea4f12.png)